src/pages/home/report/getInitialNumToRender/index.native.ts

```diff
@@ -0,0 +1,4 @@
+function getInitialNumToRender(numToRender: number): number {
+    return numToRender;
+}
+export default getInitialNumToRender;
```

Comment:
I think we usually put these functions in src/libs so I'd move it there (and then maybe document it)

Reply:
Make sense. I'll do it